src/org/gk/gkCurator/CuratorActionCollection.java

```diff
@@ -2212,13 +2214,23 @@ private void createInstance() {
         		curatorFrame.getSchemaView().setSelection(instance);
         	}
         }
-        // attribute is not editable.
+        // class is not creatable.
         else {
+        	// get contact for uncreatable classes.
+        	String contact = AttributeEditConfig.getConfig().getUncreatableClassContact();
+        	System.out.println(contact);
+
+        	// fallback if contact is not found.
+        	// this may not be necessary/useful and can be removed at your discretion.
+        	if (contact == null) {
```

Comment:
No need to check here. AttributeEditConfig has taken care of this with a default contact.

Reply:
All above reviews fixed in [239adbd](https://github.com/reactome/CuratorTool/pull/16/commits/239adbd679ef3a9c2530d4592a7fd3bd28cdd4df).